### DISCUSSION TITLE
Suchmuster Quelldateiname: Kombinationen möglich?

### DISCUSSION THREAD

**Discussion by rhinopeak** [geimist/synOCR#39] [Q&A] [answered]
Hallo,

ich habe synOCR ohne große Probleme vor kurzem installiert und es funktionierte in allen Test sehr gut. Vielen Dank für diese schöne Lösung. Ich habe zwei Fragen zu den Suchmustern:

- lassen sich im Feld Quelldateiname mehrere Suchmuster kombinieren? Z.b. "scan fromarchive$"  um nur Dateien mit dem Dateinamen "scan*fromarchive" bearbeiten zu lassen ? Oder "scan !scanocr" um die Dateien mit Dateiname scan* zu verarbeiten, aber Dateien mit Dateiname scanocr* auszuschließen? Wenn ja, wie ist die genaue Syntax (mit meinen Tests konnte ich das nicht herausfinden)?
- wenn ich "Suchmuster" als "behalten" konfigurieren - wozu dient dann der Backupordner?

Gruß

Ralf

**Comment by geimist** [comment, marked as answer]
1. Nein, Suchmuster kann man in diesem Sinn nicht kombinieren. Für unterschiedliche Szenarien sollte das Profil dupliziert werden und / oder ggf. mit einem zusätzlichen Quellordner gearbeitet werden.
2. Dieser Schalter bezieht sich auf die Umbenennung. Vielleicht definiert man im Scanner ein Präfix lediglich für das gewünschte synOCR-Profil. Diesen Präfix möchte man aber vielleicht nicht im endgültigen Dateinamen haben, wohl aber z. B. den Dateizähler des Scanners.  Lautet mein Präfix z. B. `scan_` und ich lasse das Suchmuster entfernen, so wird aus `scan_Brother 1234.pdf` eben `Brother 1234.pdf`

Ich konnte hoffentlich helfen 😊

LG Stephan

**Comment by rhinopeak** [comment]
Hallo Stephan,
vielen Dank für die super schnelle Antwort und die gute Idee.
Gruss Ralf
